src/cli.rs

```diff
@@ -22,3 +29,39 @@ impl Rover {
         }
     }
 }
+
+impl Report for Rover {
+    fn serialize_command(&self) -> Result<String, SputnikError> {
+        Ok(serde_json::to_string(&self)?)
+    }
+
+    fn is_enabled(&self) -> bool {
+        option_env!("APOLLO_TELEMETRY_DISABLED").is_none()
```

Comment:
We will need some installer disclosure (as well as docs) about this variable and our usage of telemetry

Reply:
/me nods 

ill add this as a TODO